docs/msbuild/exec-task.md

```diff
@@ -51,7 +51,7 @@ The following table describes the parameters for the `Exec` task.
 
 This task is useful when a specific MSBuild task for the job that you want to perform is not available. However, the `Exec` task, unlike a more specific task, cannot do additional processing or conditional operations based on the result of the tool or command that it runs.
 
-The `Exec` task calls *cmd.exe* instead of directly invoking a process.
+Instead of directly invoking a process, the `Exec` task calls *cmd.exe* or */bin/sh*, depending on the operating system.
```

Comment:
```suggestion
Instead of directly invoking a process, the `Exec` task calls *cmd.exe*  on Windows, or *sh* otherwise.
```

Reply:
It calls "sh" on non-Windows operating systems, but I think this would resolve to whatever sh is installed when the path is searched. It could be \bin\sh but if sh is installed somewhere else and that precedes \bin on the path, then it would be used.

Reply:
See https://github.com/dotnet/msbuild/blob/467f900dd0ae91d2ef5d3ab523cb11dbd148f320/src/Tasks/Exec.cs#L531

Reply:
@rainersigwald. We're trying to update the wording regarding the shell that is invoked by the Exec task on a non-Windows operating system.

Reply:
Your analysis seems accurate though I wonder if we _should_ call fully-qualified `/bin/sh` instead of `sh` from PATH. But I like the update with your suggestion.